data-migration/migrate-mysql-tidb-above-tb.md

```diff
@@ -0,0 +1,273 @@
+- RocksDB has a space amplification effect.
+
+## Step 1. Export all data from MySQL
+
+1. Export all data from MySQL by running the following command:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup dumpling -h ${ip} -P 3306 -u root -t 16 -r 200000 -F 256MB -B my_db1 -f 'my_db1.table[12]' -o ${data-dir}
+    ```
+
+    Dumpling exports data in SQL files by default. You can specify the file format by adding the `--filetype` option.
+
+    The parameters used above are as follows. For more Dumpling parameters, refer to [Dumpling Overview](/dumpling-overview.md).
+
+    |parameters             |Description|
+    |-                      |-|
+    |`-u` or `--user`       |MySQL user|
+    |`-p` or `--password`   |MySQL user password|
+    |`-P` or `--port`       |MySQL port|
+    |`-h` or `--host`       |MySQL IP address|
+    |`-t` or `--thread`     |The number of threads used for export|
+    |`-o` or `--output`     |The directory that stores the exported file. Supports local path or [external storage URL](/br/backup-and-restore-storages.md)|
+    |`-r` or `--row`        |The maximum number of rows in a single file|
+    |`-F`                   |The maximum size of a single file, in MiB|
+    |-`B` or `--database`   |Specifies a database to be exported|
+    |`-f` or `--filter`     |Exports tables that match the pattern. The syntax is the same as [table-filter](/table-filter.md). `[\*.\*,!/^(mysql&#124;sys&#124;INFORMATION_SCHEMA&#124;PERFORMANCE_SCHEMA&#124;METRICS_SCHEMA&#124;INSPECTION_SCHEMA)$/.\*]` exports all tables except the system schema. |
+
+    Make sure `${data-path}` has enough space to store the exported data. To prevent the import from being interrupted by a large table, it is strongly recommended to use the `-F` option to limit the size of a single file.
+
+2. View the `metadata` file in the `${data-path}` directory. This is a Dumpling-generated metadata file. Record the binlog position information, which is required for the incremental replication in Step 3.
+
+    ```
+    SHOW MASTER STATUS:
+    Log: mysql-bin.000004
+    Pos: 109227
+    GTID:
+    ```
+
+## Step 2. Import full data to TiDB
+
+1. Create the `tidb-lightning.toml` configuration file:
+
+    {{< copyable "" >}}
+
+    ```toml
+    [lightning]
+    # log.
+    level = "info"
+    file = "tidb-lightning.log"
+
+    [tikv-importer]
+    # "local": Default backend. The local backend is used to import large volumes of data (1 TiB or more). During the import, the target TiDB cluster cannot provide any service.
+    # "tidb": The "tidb" backend is used to import data less than 1 TiB. During the import, the target TiDB cluster can provide service normally. For more information on the backends, refer to https://docs.pingcap.com/tidb/stable/tidb-lightning-backends.
+    backend = "local"
+    # For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage and exclusive I/O for the directory.
```

Comment:
`For better import performance, it is recommended ....... and use flash storage, which can use I/O exclusively`